pkg/fileutil/file_exists.go

```diff
@@ -3,11 +3,23 @@
 
 package fileutil
 
-import "os"
+import (
+	"os"
+	"regexp"
+)
+
+func replaceBashVariables(input string) string {
+	r := regexp.MustCompile(`\$(\w+)|\$\{(\w+)\}`)
+	return r.ReplaceAllStringFunc(input, func(s string) string {
+		varName := r.ReplaceAllString(s, "$1$2")
+		return os.Getenv(varName)
+	})
+}
```

Comment:
These changes must be suggested in a separate PR dedicated to its purpose please.

Reply:
ok i will close this PR and open 2 new PR's